src/pybind/rados/rados.pyx

```diff
@@ -1913,6 +1920,24 @@ cdef class WriteOp(object):
         with nogil:
             rados_write_op_cmpext(self.write_op, _cmp_buf, _cmp_buf_len, _offset, NULL)
 
+    def omap_cmp(self, key: str, val: str, cmp_op: int = 1):
```

Comment:
if `LIBRADOS_CMPXATTR_OP_EQ` is available in this source file, instead of using a magic number of `1`, why not just use `LIBRADOS_CMPXATTR_OP_EQ` for better readability?

Reply:
Good point, fixed